src/core/iomgr/pollset_posix.c

```diff
@@ -161,8 +173,8 @@ void grpc_pollset_del_fd(grpc_pollset *pollset, grpc_fd *fd) {
 
 int grpc_pollset_work(grpc_pollset *pollset, gpr_timespec deadline) {
   /* pollset->mu already held */
-  gpr_timespec now;
-  now = gpr_now();
+  gpr_timespec now = gpr_now();
```

Comment:
I know the line above is valid C syntax, but I'm a little concerned that it deviates from our de-facto C style. I feel like we mostly don't call functions in initialization unless it's the only declaration, but I might be imagining this. Do we have any guidance on this?